### DISCUSSION TITLE
GitHub Achievements are a terrible feature.

### DISCUSSION THREAD

**Discussion by librity** [community/community#32352] [Other Feature Feedback, Questions, & Ideas]
GirHub Achievements are an unnecessary gamification that doesn't help asses the performance/dedication/talent/achievement of a developer.
They indicate mostly meaningless things that happen at random while using github, and most can be earned in a weekend. 
The only achievement that actually means anything is `Statstruck` because it measures something objectively good about your repos.
I wish they made achievements that show open-source contributions, or lines of code written per day.
That's something I would be glad to show other people.

**Comment by radar-blip** [comment]
You have the option to disable them if you want, in your settings.

**Reply by librity** [reply]
That's a good option. I wish they we're better though.

**Reply by radar-blip** [reply]
Well after all the feedback they received, maybe future achievements will be more meaningful. 😀

**Comment by ssteinerx** [comment]
Welcome to the gamification of everything.  

When getting stickers becomes more important than doing meaningful work. 

I want a God sticker so I can help us.
